lib/resources/r4/encounter_patch.yaml

```diff
@@ -111,3 +111,36 @@ operations:
         <li> <a href="https://fhir-ehr.cerner.com/r4/StructureDefinition/payment-collection-status?_format=json">Payment Collection Status</a></li>
         <li> <a href="https://fhir-ehr.cerner.com/r4/StructureDefinition/estimated-financial-responsibility-amount?_format=json">Estimated Financial Responsibility Amount</a></li>
       </ul>
+
+  - name: add-extension
+    path: /extension/-
+    operation: add
+    type: List of Extension
```

Comment:
This shouldn't be a list, add only supports a single extension.

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/849/commits/82c12687b37d80a0d2e24607c7931ee43adddcbb#diff-a0c53684e260623feb3826256c549a1eeb448b54febcc11c63fa0c15d4e86b20R118